### ISSUE TITLE
Unable to locate `tsserver.js` path from `mason.nvim`

### ISSUE THREAD

**Issue by shikr** [pmizio/typescript-tools.nvim#207]
I installed `typescript-language-server` from `mason.nvim`, when I open a js file I get an error:
![error](https://github.com/pmizio/typescript-tools.nvim/assets/83564960/ccb31719-acf5-4481-aaf5-c3396aa465d5)
here is my config (using lazy.nvim):
```lua
{
  'pmizio/typescript-tools.nvim',
  dependencies = 'nvim-lua/plenary.nvim',
  opts = {
    on_attach = lspconfig.on_attach,
    complete_function_calls = true,
  }
}
```
I tried using [this](https://github.com/pmizio/typescript-tools.nvim/issues/106#issuecomment-1637124589) snippet, it works, but I get the following warning:
![warn](https://github.com/pmizio/typescript-tools.nvim/assets/83564960/b1bf9dcd-f629-45cf-9e39-090e63c8a9cc)

- Is there any way to avoid displaying this warning?

**Comment by pmizio** [comment]
@shikr Sorry for delay but there was Christmas time and my vacation connected to it so...
But I checked it and I cannot reproduce this for me path resolve from mason works as expected.
Can you check this path `$MASON/packages/typescript-language-server` and give me return of `ls`.
Mason create special env variable `MASON` populated with path where all servers are installed. So you can look into it and give me more info. 
Maybe there is some edgecase which I missed.

**Comment by shikr** [comment]
I've been looking into the `MASON` variable. I'm using [NvChad](https://github.com/NvChad/NvChad), and it manually adds the path of `mason.nvim` binaries to the `PATH` variable, as shown [here](https://github.com/NvChad/NvChad/blob/v2.0/lua/core/init.lua#L59-L61). Because of this, the plugin never starts, and the `MASON` variable is never set. To fix it, I just had to load the plugin on the 'VeryLazy' event of `lazy.nvim`. Thanks for your response.

**Comment by pmizio** [comment]
Nice, thanks for response, so if it is fixed. Closing.
